routes/searchCard.js

```diff
@@ -0,0 +1,10 @@
+exports.test = function(req, res){
+  res.render('index', {
+    title: "XYZ Homeless Shelter",
+    address: "12345 Road Drive",
+    phone: "(123) 456-7890",
+    categories: ["clothing", "cleaning products", "food", "furniture"],
+    dropoff: false,
```

Comment:
rename `dropoff` and `mailin` to `isSupportDorpOff` and `isSupportMailIn` to infer the variable type to be boolean